exchanges/bitfinex/bitfinex_types.go

```diff
@@ -149,7 +149,7 @@ type Ticker struct {
 	Volume             float64
 	High               float64
 	Low                float64
-	FFRAmountAvailable float64
+	FRRAmountAvailable float64
```

Comment:
```suggestion
	FRRAmountAvailable float64 // Flash Return Rate amount available
```

Reply:
🚧  Field name is to obscure to go undocumented 🙂

Reply:
Done: https://github.com/thrasher-corp/gocryptotrader/pull/1865/commits/2f458f0ec4962a7964433200514b47cd04726d49